tutor/src/helpers/plan.coffee

```diff
@@ -27,4 +27,9 @@ PlanHelper =
     {isPublishing, publishStatus}
 
 
+  isPlanOpen: (plan) ->
+    now = moment(TimeStore.getNow())
+    !! _.find plan?.tasking_plans, (tasking) -> moment(tasking.opens_at).isBefore(now)
```

Comment:
heh, yeah. That was an oversight - I meant to remove when I switched to the `isVisibleToStudents` prop.  Thanks!